specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -24,11 +24,14 @@ export const attachmentsText = createDictionary({
     'uk-ua': 'масштаб',
   },
   attachmentServerUnavailable: {
-    'en-us': 'Attachment server unavailable.',
-    'ru-ru': 'Сервер прикрепленных файлов недоступен.',
-    'es-es': 'Servidor de archivos adjuntos no disponible.',
-    'fr-fr': 'Serveur de pièces jointes indisponible.',
-    'uk-ua': 'Сервер прикріплень недоступний.',
+    'en-us': 'Attachment server unavailable',
+    'ru-ru': 'Сервер прикрепленных файлов недоступен',
+    'es-es': 'Servidor de archivos adjuntos no disponible',
+    'fr-fr': 'Serveur de pièces jointes indisponible',
+    'uk-ua': 'Сервер прикріплень недоступний',
+  },
+  attachmentServerUnavailableDescription: {
+    'en-us': 'Setup your attachment server'
```

Comment:
I would reccomend expanding this with instructions once they are written. We can make this change at a later date

Reply:
How complex are these instructions? They may be too long to warrant putting in a dialog. 
If/when we do add (summarized?) instructions, perhaps a link to a post on the [Speciforum](https://discourse.specifysoftware.org/) may also be helpful. 
@grantfitzsimmons, Do we have public (non-member) documentation detailing the process?

Reply:
This will definitely link out to the forum! We cannot possibly localize an entire set of instructions 😂

Max asked me to write documentation for this since we do not have any user-facing docs on this. It's on my to-do list but I'll have to learn the system better myself before writing docs

Reply:
Hehe, you will become the most knowledgeable not just in web portals, but also in asset servers 😂

Reply:
I'll let you focus on the big stuff. I'm already writing an outline... now I just need to deploy some asset servers and make sure I know what's happening!